product_docs/docs/biganimal/release/free_trial/quickstart.mdx

```diff
@@ -8,56 +8,39 @@ tags:
   - psql
 ---
 
-Follow this quick walk-through to configure a PostgreSQL 15 cluster on BigAnimal running against AWS with a free trial account. 
+Follow this quick walk-through to configure a Postgres cluster on BigAnimal. 
 
 ## Step 1: Create an account and sign into the portal
 
 If you haven't done so already, you'll need to [create your EDB account](detail/create_an_account). 
 
-Then, use your newly created account to access the [BigAnimal free trial portal](https://freetrial.biganimal.com/).
+Then, use your newly created account to access the [BigAnimal](https://portal.biganimal.com) portal.
 
-!!! Note
-    Continue to use the portal at [freetrial.biganimal.com](https://freetrial.biganimal.com/) to access BigAnimal  for the duration of your free trial.
 
-## Step 2: Create an AWS cluster 
+## Step 2: Create a cluster 
 
 1. Select the **Clusters** link on the left to navigate to the [Clusters](https://portal.biganimal.com/clusters) page.
 
 1. Select **Create New Cluster**.
 
     You should now find yourself at the [Create Cluster page](https://portal.biganimal.com/create-cluster).
 
-1. Select **Single Node** as your cluster type.
-1. For **Provider**, select **AWS**.
-1. Select **Next: Cluster Settings**.
-1. In the **Cluster Name** field, enter `AWS PostgreSQL cluster`.
-1. In the **Password** field, enter a strong, memorable password. 
-
-   This is the admin password for the cluster.
-1. For **Database Type**, select **PostgreSQL**. 
-
-   This selection gives you an official build of PostgreSQL.
-1. For **Postgres Version**, select **15**.
-
-1. For **Region**, select **US East 1**.
-1. For **Instance Type** and **Storage**, only one set of options is available in the trial. Select each option.
-1. Select **Create Cluster**. You'll be brought back to the Clusters page with your newly configured cluster now populating the list. It usually takes a minute or two for your cluster to be ready but can take up to an hour. A progress bar is shown near the right.
+1. Select the options for your cluster. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/) for more information.
 
 ### Managing your cluster
 
-After you select **Create Cluster**, you'll be brought back to the Clusters page with your newly configured cluster now populating the list. In the free trial, you can provision only one cluster at a time. That said, you can delete your cluster, start a new cluster, and even restore a deleted cluster. BigAnimal automatically backs up clusters.
-
-Once your first cluster is fully provisioned, your 14-day free trial officially starts, and a countdown appears telling you how many days are left in your trial.
+After you select **Create Cluster**, you'll be brought back to the Clusters page with your newly configured cluster now populating the list. 
```

Comment:
Should this part mention what users can do in terms of cluster management? i.e. edit cluster, create faraway replica, etc.

Reply:
done